docs/intelligentapps/bulkrun.md

```diff
@@ -0,0 +1,42 @@
+---
+Order: 4
+Area: intelligentapps
+TOCTitle: Bulk Run
+ContentId:
+PageTitle: Bulk Run Prompts
+DateApproved:
+MetaDescription: Run a set of prompts in an imported dataset, individually or in a full batch towards the selected genAI models and parameters.
+MetaSocialImage:
+---
+
+# Bulk Run Overview
+
+Bulk run feature in AI Toolkit allows you to run many prompts in batch mode. Compared to playground you can only run each prompt manually in order.
+
+## Start a bulk run
+
+Navigate to `Bulk Run` in the `TOOLS` section in the tree view of AI Toolkit. You can select either a sample dataset to get familiar about the feature, or import a local JSONL file that has at least a field `query` to use as prompts.
```

Comment:
Use numbered steps to make it easier for users to know what to do. Also include screenshots where needed.